topics/galaxy-project/metadata.yaml

```diff
@@ -0,0 +1,19 @@
+---
+name: "galaxy-project"
+type: "use"
+enable: false
+title: "Galaxy project guide"
+summary: "Galaxy is an unusual open source project. Since starting in 2005, the project code has been deployed continuously, on increasingly substantial computational infrastructure, to provide accessible, reliable and transparent open science data analysis and training at no cost to researchers. 
+
+These highly valued and reliable services operate 24x7, supported by multiple collaborating grants and a large community of contributors. 
+
+Open science services are a distinguishing feature, compared to most other self-governing open source projects. Comparable, multi-disciplinary analysis services at scale, at no cost to researchers, are rare and few are as heavily used. Comparable commercial services are extremely costly, and may not be transparent or readily shared.
+
+How is it possible for Galaxy to sustain high quality, free services? What makes Galaxy so valuable to scientists? Are there lessons to be learned in the story of the origins, growth and success?
+
+These are some of the issues addressed in this training topic, in describing the major components and activities of the project that sustains Galaxy.
+Extremely Work in progress November 2022."
```

Comment:
```suggestion
summary: |
  Galaxy is an unusual open source project. Since starting in 2005, the project code has been deployed continuously, on increasingly substantial computational infrastructure, to provide accessible, reliable and transparent open science data analysis and training at no cost to researchers. 

  These highly valued and reliable services operate 24x7, supported by multiple collaborating grants and a large community of contributors. 

  Open science services are a distinguishing feature, compared to most other self-governing open source projects. Comparable, multi-disciplinary analysis services at scale, at no cost to researchers, are rare and few are as heavily used. Comparable commercial services are extremely costly, and may not be transparent or readily shared.

  How is it possible for Galaxy to sustain high quality, free services? What makes Galaxy so valuable to scientists? Are there lessons to be learned in the story of the origins, growth and success?

 These are some of the issues addressed in this training topic, in describing the major components and activities of the project that sustains Galaxy.
  Extremely Work in progress November 2022.
```

just nicer to read